PointOfDispensingApp/config.js

```diff
@@ -0,0 +1,10 @@
+//file name is 'config.js'
+
+export default {
+    vaccinationDecisionState:
```

Comment:
This name doesn't match the actual enums you're storing. Maybe `CanProceedWithVaccination`? That then begs the question of why not use a bool and assume always `Vaccination_Proceed_No` until it becomes true?

Reply:
There are 3 states of the UI presentation:
Initial state (no decision has been made as to whether or not vaccine will be administered):  we shouldn't display the specific fields for documenting the vaccination (for a decision to proceed with the vaccination) and we shouldn't display the specific fields for documenting why the vaccine will not be administered (for a decision to cancel the vaccination).
I am using "vaccinationDecisionState" to capture the state of the user's decision (0. no decision has been made, yet; 1. decision has been made to proceed with vaccination; 2. decision has been made to cancel the vaccination).

Does that help?  Does that make more sense?